src/vector.rs

```diff
@@ -51,7 +54,20 @@ pub trait Vector {
     /// Returns vector graphics that implements [`kurbo::Shape`].
     fn shape(&self) -> impl kurbo::Shape;
     /// Translation of the border at a specific `time` value.
-    fn border_translation(&self, time: f64) -> DVec2;
+    fn border_translation(&self, time: f64) -> DVec2 {
+        let path = BezPath::from_iter(self.shape().path_elements(0.0));
+
+        VelloBezPath::default()
+            .with_path(path)
+            .border_translation(time)
+    }
+
     /// The rotation at the tangent of the border at a specific `time` value.
-    fn border_rotation(&self, time: f64) -> f64;
+    fn border_rotation(&self, time: f64) -> f64 {
+        let path = BezPath::from_iter(self.shape().path_elements(0.0));
+
+        VelloBezPath::default()
+            .with_path(path)
+            .border_rotation(time)
+    }
```

Comment:
not sure if this is a good idea haha because specialized functions are always more efficient IMO. But also in a way it's kinda cool to just have a default implementation without the need to do anything extra when adding new shapes haha.

Reply:
i agree lol. fixed in [861dd7f](https://github.com/voxell-tech/bevy_vello_graphics/pull/3/commits/861dd7f657aaabe0aea7a6f39a464ef6cef48ca3)

Reply:
oops I think I may have misinterpreted what you meant. 

I thought you meant the generic implementation of bezpath `border_rotation` which was fixed in that commit 

but now I'm thinking that you meant the default implementation of the `border*` functions which we could simply replace with `DVec::ZERO` and `0.0` respectively much like how we do it for rectangle and circle shape